TAF/testScenarios/functionalTest/V2-API/core-metadata/deviceprofile/DELETE-Negative-Deviceresource.robot

```diff
@@ -0,0 +1,39 @@
+*** Settings ***
+Library      uuid
+Resource     TAF/testCaseModules/keywords/common/commonKeywords.robot
+Resource     TAF/testCaseModules/keywords/core-metadata/coreMetadataAPI.robot
+Suite Setup  Run Keywords  Setup Suite
+...                        AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
+Suite Teardown  Run Teardown Keywords
+Force Tags      Skipped
+
+*** Variables ***
+${SUITE}          Core Metadata Device Profile DELETE Device Resource Negative Test Cases
+${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/core-metadata-deviceprofile-delete-deviceresource-negative.log
+
+*** Test Cases ***
+ErrProfileResourceDELETE001 - Delete deviceResource by non-existent deviceResource name
+    # non-existent deviceResource name
+    Given Create A Device Profile Sample With non-existent deviceResource name
+    When Delete deviceResource By Name
```

Comment:
Should create a normal device profile and delete device resource by non-existent deviceResource name